zip-0032.rst

```diff
@@ -442,6 +442,9 @@ they MAY additionally support a non-hardened :math:`address\_index` path level a
 
 * :math:`m_\mathsf{Sapling} / purpose' / coin\_type' / account' / address\_index`.
 
+`zcashd` version 4.5.2 and later uses this to derive "legacy" Sapling addresses from a mnemonic seed phrase
+under account :math:`\mathtt{0x7FFFFFFF}`.
```

Comment:
We should note also that zcashd uses hardened derivation for `address_index` in this usage pattern.

Reply:
```suggestion
under account :math:`\mathtt{0x7FFFFFFF}`, using hardened derivation for :math:`address\_index`.
```